### ISSUE TITLE
Wrong IVs calculations.

### ISSUE THREAD

**Issue by mattaereal** [farkam135/GoIV#173]
I have an Eevee with 507 CP, 67 HP. It requires 2200 Stardust & 2 Candy to power up.

GoIV says my Eevee is 100% IV, perfect stats 15/15/15. BUT the [new appraisal system](http://www.pokemongodb.net/2016/08/appraise.html) shows that my Eevee is only 15 on ATK , and on the rest lower than 15. Narrowing it down to the possibility being level 17, 15/10/12 and 82.2% IVs.

If you're reading this and don't understand about the appraisal system, check out [this guide.](https://imgur.com/a/zofAR#cXLkgyY)

The bottom line is that Niantic's appraisal system don't miss. It said I had only 15 on ATK.

**UPDATE:** Confirmed that the calculations have nothing to do with this issue at least. It's arc misalignment.

**Comment by Blaisorblade** [comment]
I don't know the IV formulas so I can't vouch for their correctness, but could GoIV have guessed a level that's too low?

**Comment by rpgrca** [comment]
If you could post a screenshot, as @Blaisorblade said, it looks like the dot wasn't correctly set.

**Comment by mattaereal** [comment]
Yeah, that happened. It's arc misalignment related. Just confirmed it. I had to move the red dot one step away from the white dot.

**Comment by mattaereal** [comment]
This are the screens before the last update. If with the latest version that I'm currently building I notice a difference I'll reupload the screenies.

![image](https://cloud.githubusercontent.com/assets/388605/17940980/9d5898f2-6a31-11e6-9c14-653caf658dc9.png)

![image](https://cloud.githubusercontent.com/assets/388605/17941055/fbbbb938-6a31-11e6-8cd3-9ec19e91c7c8.png)

![image](https://cloud.githubusercontent.com/assets/388605/17941080/0c6a2242-6a32-11e6-8e44-5910a20d85c6.png)


![image](https://cloud.githubusercontent.com/assets/388605/17941170/70a99a08-6a32-11e6-83c3-51eef8ea8371.png)

**Comment by rpgrca** [comment]
And you have your level set correctly too, right?

**Comment by mattaereal** [comment]
Nevermind, I was one pokemon away to level up, and accidentally leveled up while testing the app. Sorry guys :-1:
